src/server/rdb_test.cc

```diff
@@ -54,6 +54,7 @@ class RdbTest : public BaseFamilyTest {
 
 void RdbTest::SetUp() {
   InitWithDbFilename();
+  used_mem_current = 0;
```

Comment:
This is because of the rdb failures right ?

Reply:
this indeed seems unrelated to this PR :)